ETC_en.tsv

```diff
@@ -5503,13 +5503,13 @@ ETC_20150317_005502	Defeated 500 boss monsters.
 ETC_20150317_005503	Walking Dead
 ETC_20150317_005504	Achieved by resurrecting at least 100 times using NXP.
 ETC_20150317_005505	Crafting Beginner
-ETC_20150317_005506	Succeeded in crafting 10 times. Feels pretty good.
```

Comment:
I don't know. "crafting an item 10 times" sounds almost like you have to craft the same item 10 times. "by crafting 10 items" is simple and straightforward for the player, imo.

Thanks a lot for the notes, I have changed them all except this one. Will try to cross check the original Korean text next time. :)